### PR TITLE
feat: [CDS-42901]: Freeze windows List view changes

### PR DESCRIPTION
**Pull request by ishantmehtaharness**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Passed_** 
 


            
 No issues Everything looks good
 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/11172)

**Review by udayvunnam — DISMISSED** [review]

File: src/modules/72-freeze-windows/components/FreezeWindowStudio/FreezeWindowStudioHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import type { ModulePathParams, ProjectPathProps } from '@common/interfaces/RouteInterfaces'
+
+export const FreezeWindowStudioHeader = () => {
+  const { getString } = useStrings()
+  const { projectIdentifier, orgIdentifier, accountId, module } = useParams<ProjectPathProps & ModulePathParams>()
+
+  return (
+    <Page.Header
+      size={'small'}
+      title={null}
```
Comment (udayvunnam) [inline src/modules/72-freeze-windows/components/FreezeWindowStudio/FreezeWindowStudioHeader.tsx:23]: Will the title be decided later?
nit: size="small"

Comment (ishantmehtaharness) [reply src/modules/72-freeze-windows/components/FreezeWindowStudio/FreezeWindowStudioHeader.tsx:23]: Yes, title will be decided in subsequent PR's

File: src/modules/72-freeze-windows/components/FreezeWindowStudio/FreezeWindowStudioSubHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+        {/*{modalProps && <TemplateConfigModalWithRef {...modalProps} onClose={onCloseCreate} />}*/}
+      </Dialog>
+    )
+  }, [windowIdentifier])
+
+  React.useEffect(() => {
+    if (windowIdentifier === '-1') {
+      hideConfigModal()
+      showConfigModal()
+    }
```
Comment (udayvunnam) [inline src/modules/72-freeze-windows/components/FreezeWindowStudio/FreezeWindowStudioSubHeader.tsx:74]: we should find a better way to handle this.

**Review by udayvunnam — APPROVED** [review]

**Review by kapilkumawat86 — APPROVED** [review]

**Pull request merged**